sklearn/model_selection/_validation.py

```diff
@@ -265,7 +265,8 @@ def _fit_and_score(estimator, X, y, scorer, train, test, verbose,
     if verbose > 2:
         msg += ", score=%f" % test_score
     if verbose > 1:
-        end_msg = "%s -%s" % (msg, logger.short_format_time(score_time))
+        total_time = score_time + fit_time
+        end_msg = "%s -%s" % (msg, logger.short_format_time(total_time))
```

Comment:
shouldn't it say total time? or fit time and score time separately? At least ``total: %s`` maybe?